src/augmentations.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import random
+import cv2
+
+def to_numpy_img(img):
+    """From tensor to numpy, with channels last"""
+
+    img = img.data.numpy()
+    img = np.moveaxis(img, 0, -1)
+
+    return img
+
+def rotate(img, mask, rot=(-10, 10), p=0.5):
+    """Rotate image and crossponding mask with the same random angle"""
+
+    if random.random() > p:
```

Comment:
Isn't this a sort of unintuitive definition of p? I feel like people would assume it's the rotation probability, but isn't this 1 - the augmentation probability?

Reply:
It's the rotation probability. There's no augmentation probability, every operation has its own p, for no augmentation, one needs to set all to 0.
Can add another parameter to set all to 0, when augmentation=false

Reply:
Sorry, I meant rotation probability. But what I mean is that, if I set p = 0.9 in the argument to this function, won't this return a rotated result with probability 0.1?

Reply:
Got you. You're right, it's toggled, will make it the other way around